### PR TITLE
BLadd24994

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

File: LondonBritishLibrary/add/BLadd24994.xml [hunk 1]
```diff
@@ -42,40 +42,44 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                             </msItem>
                         
-                        
                         <msItem xml:id="ms_i3">
                             <locus from="10v"/>
                             <locus from="19r"/>
-                            <!--<title type="complete" ref=""/>-->
+                            <title type="complete" ref="LIT6628Sabbath"/>
                             <textLang mainLang="gez" otherLangs="am"/>
                             <incipit xml:lang="gez">ወነበቦ፡ እግዚአብሔር፡ ለሙሴ፡ ወይቤሎ፡ ንግሮሙ፡ ለደቂቀ፡ እስራኤል፡ ወበሎሙ፡ ወአንተኒ፡ አዝዞሙ፡
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/add/BLadd24994.xml:78]: ንገሮሙ?

File: LondonBritishLibrary/add/BLadd24994.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title/>
+                <title>Apocalyptic Beta ʾƎsrāʾel texts, Canticles</title>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/add/BLadd24994.xml:8]: Sorry, I miss Canticles, what is this?

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/add/BLadd24994.xml (hunk 1 above)
Comment (thea-m) [reply LondonBritishLibrary/add/BLadd24994.xml:78]: I think this is the regular imperative.

File: LondonBritishLibrary/add/BLadd24994.xml (hunk 2 above)
Comment (thea-m) [reply LondonBritishLibrary/add/BLadd24994.xml:8]: Not sure I understand what you mean :) The Biblical canticles, starting from f. 15v here

File: LondonBritishLibrary/add/BLadd24994.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/add/BLadd24994.xml:78]: Indeed; there are two forms possible.

**Pull request merged**